assets/css/style.css

```diff
@@ -1,111 +1,136 @@
-    flex-direction: row-reverse;
-    margin-top: 5%;
-    height: 5%;
+    flex-direction: row;
+    justify-content: center;
+    height: 72px;
+    margin-top: 1%;
   }
 
   #hamburg-icon {
-    font-size: 17pt;
+    font-size: 17px;
     padding-right: 2rem;
     color: #28352f;
+    display: none;
   }
 
-  main {
-    background-image: url(../img/header-illsutration-mobile@2x.png);
-    background-size: 100%;
-    background-repeat: no-repeat;
-    margin-top: 5%;
+  nav ul {
+    display: flex;
+    justify-content: center;
+    height: 40px;
+    gap: 25px;
+  }
+
+  nav ul li {
+    list-style: none;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
   }
 
+  main { margin-top: 5%; }
+
   .socialmedia-links {
     display: flex;
     flex-direction: row;
-    margin-top: 6%;
     justify-content: center;
-    transform: translateX(-7%);
-    gap: 1.5%;
+    align-items: center;
+    margin-left: -2.5%;
   }
 
   .socialmedia-links li {
     list-style: none;
-    padding-left: 4%;
+    color: #505f79;
+    padding: 8px;
+    width: 40px;
+    height: 40px;
   }
 
-  .socialmedia-links img {
-    height: 90%;
-    width: 90%;
+  .headline-section {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    margin-left: 18%;
+    margin-right: 18%;
+    margin-top: 5%;
   }
 
-  .headline-section { margin-top: 20%; }
-
   .headline-primary,
   .headline-secondary {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 800;
+    font-size: 48px;
+    line-height: 60px;
+    text-align: center;
+    letter-spacing: 0.37px;
+  }
+
+  .headline-container h2 {
     font-family: 'Inter', sans-serif;
     font-style: normal;
     font-weight: 700;
     font-size: 40px;
     line-height: 52px;
     text-align: center;
+    color: #172b4d;
   }
 
   .headline-primary { color: #172b4d; }
 
   .headline-secondary { color: #36b37e; }
 
   .headline-content p {
-    margin-top: -8%;
     font-family: 'Inter', sans-serif;
     font-style: normal;
     font-weight: 400;
-    font-size: 18px;
-    line-height: 26px;
+    font-size: 20px;
+    line-height: 32px;
     text-align: center;
     color: #172b4d;
-    padding-left: 0.5%;
-    padding-right: 0.5%;
+    padding-left: 5%;
+    padding-right: 5%;
   }
 
-  .headline-content {
-    margin-left: 5%;
-    margin-right: 5%;
-    padding-left: 3%;
-    padding-right: 3%;
-  }
+  .headline-content { margin-top: -5%; }
 
   .btn-hover {
-    text-align: center;
-    padding-top: 15%;
+    display: flex;
+    justify-content: center;
+    margin-top: 5%;
+    align-items: center;
+    width: 100%;
   }
 
-  .contact-container { margin-top: 10%; }
-
   .indicator {
     width: 48px;
     height: 4px;
     background: #36b37e;
     border-radius: 24px;
     margin-top: -5%;
+    display: none;
   }
 
   .works-section {
```

Comment:
- [ ] For the works sections, some of the project cards are hidden and the title is also not centered. Please note that we want to get a design close to the Figma template as much as possible according to the project requirements  Kindly fix this⛏🤓 See the video below for more clarity 👇

https://github.com/Graciano1997/portofolio-graciano/assets/106119105/f445a17f-cc65-4834-a62a-07a608ee48d7